buildSrc/build.gradle

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * This configuration specifies gradle plugins for all Google Ads Java library
+ * subprojects.
+ */
+
+plugins {
```

Comment:
Curious to hear why this would not just be in the root directory of the project. Is `buildSrc` a common convention?

Reply:
This is the recommended structure for a project with multiple subprojects (as we have here). `buildSrc` is the right place for build logic that applies to the whole project. For example, this might be the appropriate place to add Maven repository release logic/configuration.

You _could_ have a gradle.build file in the root directory with much of these contents, but the `buildSrc` directory allows for greater flexibility, tests, etc. There's a bit of explanation here: https://docs.gradle.org/current/userguide/organizing_gradle_projects.html#sec:build_sources.